packages/flutter/lib/src/material/date_picker.dart

```diff
@@ -17,6 +17,16 @@ import 'typography.dart';
 
```

Comment:
for all the DateTime fields in this file, mention what time zone is used to select the date. Local time zone? UTC? The one that the DateTime specifies?